### ISSUE TITLE
Closing patch needs to refresh grid

### ISSUE THREAD

**Issue by AZHenley** [utk-se/CodeRibbon#51]
Closing a patch causes that row to be offset.

**Comment by robobenklein** [comment]
Probably found the deal for removing `contextMenuItem`s

```js
atom.contextMenu.itemSets.find(e => { return e.items.find(i => { return i.label == "Close Pane" } })

// for only the others:
atom.contextMenu.itemSets.filter(function(val, index, arr) { return val.items.find(i => { return i.label == "Close Pane" } ) == undefined } )
```

This would just remove the menu item, not prevent closing patches since I don't immediately know of a good way to differentiate between 'close pane' and the destroy() call.

**Comment by robobenklein** [comment]
In the patch deconstructor: global_cr_update()

But probably don't want to have this run multiples for trim operations, but would also be a fix for too many empty patches, though an itemRemoved() with moved=false update sounds better there.

**Comment by robobenklein** [comment]
completely solved by #59 and #52
